packages/bsky/tests/admin/__snapshots__/get-record.test.ts.snap

```diff
@@ -1,18 +1,28 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`pds admin get record view gets a record by uri and cid. 1`] = `
+exports[`admin get record view gets a record by uri and cid. 1`] = `
 Object {
   "blobCids": Array [],
   "blobs": Array [],
   "cid": "cids(0)",
   "indexedAt": "1970-01-01T00:00:00.000Z",
+  "labels": Array [
+    Object {
+      "cid": "cids(0)",
+      "cts": "1970-01-01T00:00:00.000Z",
+      "neg": false,
+      "src": "user(0)",
+      "uri": "record(0)",
+      "val": "self-label",
+    },
+  ],
```

Comment:
Is it easy for us to account for this self-label now appearing here?

Reply:
Yup the view is fulfilled by the appview now. The moderation views service on the PDS didn't hydrate in self labels 👍